frappe/build.py

```diff
@@ -76,27 +193,28 @@ def setup():
 
 
 def get_node_pacman():
-	pacmans = ['yarn', 'npm']
-	for exec_ in pacmans:
-		exec_ = find_executable(exec_)
-		if exec_:
-			return exec_
-	raise ValueError('No Node.js Package Manager found.')
+	exec_ = find_executable("yarn")
```

Comment:
Any reason to explicitly use Yarn?

Reply:
We've de-listed it as a Frappe dependency for quite some time. Even `npm install` triggers `yarn run snyk-protect`. 

Any reason to keep `npm` support around?

Reply:
> Any reason to keep npm support around?

None so far, I guess it's fine